workflow/scripts/collect_rvr_measurements.py

```diff
@@ -0,0 +1,136 @@
+            The pyRadiomics feature dataframe you would like to obtain a subset of data from
+        diams: str 
+            Decides which set of diameter measurements is used
+                max_diam --> Maximum 2D diameters (OLD, DO NOT USE)
+                axis_len --> Axis lengths 
+        
+        Returns 
+        ----------
+        feat_w_tva_df: pd.DataFrame 
+            The original dataframe with the addition of a column containing the TVA values for all patients
+        '''
+        if diams == "max_diam":
+            #This calculation is not for an encompassing ellipsoid. Will likely get removed later
+            feat_df["original_shape_gordon_tva"] = 4/3*np.pi*feat_df["original_shape_Maximum2DDiameterRow"]*feat_df["original_shape_Maximum2DDiameterColumn"]*feat_df["original_shape_Maximum2DDiameterSlice"]
+        elif diams == "axis_len":
+            feat_df["original_shape_gordon_tva"] = 4/3*np.pi*feat_df["original_shape_MajorAxisLength"]/2*feat_df["original_shape_MinorAxisLength"]/2*feat_df["original_shape_LeastAxisLength"]/2 #Divide to get semi-axis lengths
+        else: 
+            raise ValueError("Diameter setting not accepted. Please choose either max_diam or axis_len.")
+        feat_w_tva_df = feat_df.copy(deep = True) 
+
+        return feat_w_tva_df
+
+    feat_df_w_tva = calc_gordon_tva(feature_df, diams = diameters)
+    patient_file_info = get_pat_and_file_info(feature_df)
+
+    diagnostic_labels = ["diagnostics_Image-original_Spacing", 
+                         "diagnostics_Image-interpolated_Spacing",
+                         "diagnostics_Image-original_Size", 
+                         "diagnostics_Image-interpolated_Size"
+                        ]
+    
+    feature_labels = ["original_shape_MeshVolume", 
+                      "original_shape_VoxelVolume", 
+                      "original_shape_Maximum3DDiameter", 
+                      "original_shape_Maximum2DDiameterColumn", 
+                      "original_shape_Maximum2DDiameterRow", 
+                      "original_shape_Maximum2DDiameterSlice", 
+                      "original_shape_MajorAxisLength",
+                      "original_shape_MinorAxisLength",
+                      "original_shape_LeastAxisLength",
+                      "original_shape_gordon_tva"
+                     ]
+    
+    labels_of_interest = diagnostic_labels + feature_labels
+
+    diag_feat_df = feat_df_w_tva[labels_of_interest]
+
+    pat_diag_feat_df = pd.concat([patient_file_info, diag_feat_df], axis = 1)
+
+    return pat_diag_feat_df 
+
+@click.command()
+@click.option('--feat_path', help = 'Path to pyradiomics feature .csv file')
+@click.option('--export_path', help = 'Path to export directory')
+@click.option('--export_filename', help = 'Name of consolidated feature .csv file')
+def create_rvr_measurements(feat_path: str, 
+                            export_path: str,
+                            export_filename: str): 
```

Comment:
Updates to go with my comment below about default filepaths

```suggestion
def create_rvr_measurements(feat_path: str,
                            dataset_name: str,
                            export_path: str = None,
                            export_filename: str = None): 
```